.github/workflows/dev_labeler/labeler.yml

```diff
@@ -16,7 +16,9 @@
 # under the License.
 
 rust-lang:
-- any: ['rust/*']
+  - rust/*
```

Comment:
Do we need this?
Could you confirm that `rust/**/*` covers `rust/*`?

Reply:
I confirmed during testing on my fork that both are required. This is also a [common example](https://github.com/marketplace/actions/labeler#common-examples), the `@domain/core` example, on the README.

Reply:
Thanks.
I also confirmed it with:

  * https://github.com/kou/arrow/commit/7f37e3566d9fb61287777c0e5fa83c0fd13f0b64
  * https://github.com/kou/arrow/pull/7

The `rust-lang` tag was assigned for `rust/README.md` change without `rust/*`.

It seems that `rust/*` is needless.

Reply:
Thanks @kou . I probably made a mistake during testing in my repo, then. If it works, it works 👍 I removed it.